Build fails with 501

maven build fails with 

`[ERROR] Failed to execute goal on project semagrow-monitor: Could not resolve dependencies for project org.semagrow:semagrow-monitor:jar:2.1-SNAPSHOT: Could not transfer artifact org.semagrow:semagrow-commons:jar:2.1-SNAPSHOT from/to maven-central (http://repo1.maven.org/maven2): Failed to transfer file: http://repo1.maven.org/maven2/org/semagrow/semagrow-commons/2.1-SNAPSHOT/semagrow-commons-2.1-SNAPSHOT.jar. Return code is: 501 , ReasonPhrase:HTTPS Required. -> [Help 1]`

This is related to enforced HTTPS from maven central since January 15 2020 https://blog.sonatype.com/central-repository-moving-to-https

--- a/pom.xml
+++ b/pom.xml
@@ -96,7 +96,7 @@
     <repositories>
         <repository>
             <id>maven-central</id>
-            <url>http://repo1.maven.org/maven2</url>
+            <url>https://repo1.maven.org/maven2/</url>
         </repository>
         <repository>
             <id>spring-milestone</id>

